docs/pkg.conf.5

```diff
@@ -122,16 +122,16 @@ Default: NO.
 .It Cm BACKUP_LIBRARIES: boolean
 If set to
 .Sy true
+and if an upgrade will remove a library, then
 .Xr pkg 8
-will backup ancient libraries if they are removed as the result of an upgrade
-and keep a copy in the path define in
+will backup the library to the path defined by
 .Cm BACKUP_LIBRARY_PATH .
-If it does not exist yet a new package
+An initial backup will create the 
 .Qq compat-libraries
-will be created. the version will be bumped each time a new library is backed up
+package. The package version will be bumped whenever an additional library is backed up.
```

Comment:
```suggestion
package.
The package version will be bumped whenever an additional library is
backed up.
```

Sorry to backtrack, I forgot something! Roff requires one sentence per line, and please wrap at 72 for compatibility with standardized methodology and tooling in base. After that, LGTM!